src/lv_widgets/lv_cont.c

```diff
@@ -649,7 +649,6 @@ static void lv_cont_layout_pretty(lv_obj_t * cont)
  */
 static void lv_cont_layout_grid(lv_obj_t * cont)
 {
-
```

Comment:
Is this a leftover from past merge conflicts?

Reply:
Do you mean the removed empty line?
If so, honestly, I don't know. :) Maybe I just saw the empty line while integrating this feature and removed it.